ee/tabby-webserver/src/service/auth.rs

```diff
@@ -233,6 +236,12 @@ impl AuthenticationService for AuthenticationServiceImpl {
     }
 
     async fn create_invitation(&self, email: String) -> Result<Invitation> {
+        if !self.license.read_license().await.is_license_valid() {
```

Comment:
Do we want to just make a `.is_license_valid()` function on `LicenseService`?

Reply:
nope - to ensure we always have full license info at the place applying license check.